examples/gno.land/r/gov/dao/v3/memberstore/memberstore_test.gno

```diff
@@ -115,7 +114,7 @@ func TestCreateMembers(t *testing.T) {
 }
 
 func addMembers(ms MembersByTier, c int, tier string) {
-	mt := avl.NewTree()
+	// mt := avl.NewTree() XXX
```

Comment:
?

Reply:
examples *_test.gno files were never type-checked, and mt isn't used.